Misc/NEWS.d/next/Library/2018-07-11-20-51-20.bpo-34070.WpmFAu.rst

```diff
@@ -0,0 +1,2 @@
+Make sure to only check if the handle is a tty, when opening
+a file with `buffering=-1`.
```

Comment:
Use double backquotes:
```rst
``buffering=-1``
```
This is a reStructuredText, not Markdown.

Reply:
Done!

Sorry about that, haven't been doing enough Python recently :(